### ISSUE TITLE
SMS notification is sent for some case. 

### ISSUE THREAD

**Issue by GochaLearningSpace** [Willings-Inc/SCG#168]

**Comment by GochaLearningSpace** [comment]
https://user-images.githubusercontent.com/6176526/236626521-dd10c9a1-0513-4060-8d6c-4712e2e19c10.MP4

i phone safari, email login,

- forget inputting email 
- retry
- SMS sent
- SMS is not received.

**Comment by Sau1503k** [comment]
@GochaLearningSpace Now in the new update we have removed phone registration feature.

**Comment by Sau1503k** [comment]
@GochaLearningSpace, yes we save the number in the backend but don't use it for the authentication
We only verify email address and user can register only through email id
